<launch>

	<!-- <param name="use_sim_time" value="true" />

	<node name="Stage" pkg="stage_ros" type="stageros" args="$(find robodog)/world/office.world">
		<param name="base_watchdog_timeout" value="0" />
	</node> -->
	<!-- <node name="R0_MapAlign" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 /map /robot_0/map 100"/> -->

<group ns="robot_0">
    <param name="robot_id" value="1" />
    <param name="tf_prefix" type="string" value="robot_0"/>
    <rosparam file="$(find robodog)/param/ros.yaml"/>
    <node pkg="nav2d_karto" name="Mapper" type="mapper"/>
	<node pkg="robodog" name="leader_follower" type="leader_follower.py"/>
</group>

<group ns="robot_1">
    <param name="robot_id" value="2" />
    <param name="tf_prefix" type="string" value="robot_1"/>
    <node pkg="robodog" name="walk" type="walk.py">
        <param name="robot_id" value="1"/>
        <param name="other_robot" value="2"/>
        <param name="linear_velocity" value="0.8"/>
         <param name="angular_velocity" value="0.0"/>
    </node>
</group>
<group ns="robot_2">
    <param name="robot_id" value="3" />
    <param name="tf_prefix" type="string" value="robot_3"/>
    <node pkg="robodog" name="walk" type="walk.py">
        <param name="robot_id" value="2"/>
        <param name="other_robot" value="1"/>
         <param name="angular_velocity" value="0.0"/>
        <param name="linear_velocity" value="0.8"/>
    </node>
</group>

<!-- <node name="RVIZ" pkg="rviz" type="rviz" args=" -d $(find robodog)/param/robodog.rviz" output="screen" /> -->

</launch>
